orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/dcos/DcosContainerFinder.groovy

```diff
@@ -20,7 +20,10 @@ class DcosContainerFinder {
       if (!image) {
         throw new IllegalStateException("No image found in context for pattern $imageDescription.pattern.")
       } else {
-        imageDescription = [registry: image.registry, tag: image.tag, repository: image.repository, imageId: buildImageId(image.registry, image.repository, image.tag)]
```

Comment:
I don't know what was going on, maybe it was trying to reassign to that new map rather than merging them? All I know is after this change the imageId field was actually being populated correctly compared to before that.